src/Classes/ServiceAPI/MyRadio_Season.php

```diff
@@ -920,7 +920,7 @@ public function schedule($params)
                 $day_start = $start_day + (($i - 1) * 7 * 86400);
                 $show_time = $day_start + $req_time['start_time'];
 
-                $conflict = MyRadio_Scheduler::getScheduleConflict($day_start + $req_time['start_time'], $day_start + $start_time + $req_time['duration'] - 1);
+                $conflict = MyRadio_Scheduler::getScheduleConflict($show_time, $show_time + $req_time['duration']);
```

Comment:
the -1 is done in MyRadio_Scheduler::getScheduleConflict so it seemed like it was being applied twice